project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}plan.tex

```diff
@@ -0,0 +1,44 @@
+\documentclass[DM,lsstdraft,toc]{lsstdoc}
```

Comment:
Double check the default subsystem be `SE`?

Reply:
i did not change it